test/cancel_test.go

```diff
@@ -136,7 +134,7 @@ func TestTaskRunPipelineRunCancel(t *testing.T) {
 		t.Errorf("Error waiting for PipelineRun `pear` to finished: %s", err)
 	}
 
-	logger.Infof("Waiting for TaskRuns in PipelineRun %s in namespace %s to be cancelled", "pear", namespace)
+	t.Logf("Waiting for TaskRuns in PipelineRun %s in namespace %s to be cancelled", "pear", namespace)
```

Comment:
there's actually one problem with using `t.Logf` in these integration tests: afaik the output is buffered, so if you run the test locally, you won't see the output until the test is done (e.g. maybe 5+ min later) :S